pettingzoo/utils/wrappers/base.py

```diff
@@ -98,13 +102,13 @@ def reset(self, seed=None, options=None):
         self.agents = self.env.agents
         self._cumulative_rewards = self.env._cumulative_rewards
 
-    def observe(self, agent):
+    def observe(self, agent: str) -> ObsType | None:
         return self.env.observe(agent)
 
-    def state(self):
+    def state(self) -> np.ndarray:
```

Comment:
Not sure if we restrict state to be a given type, it seems like we should let users do what they want

Reply:
@jjshoots @RedTachyon any thoughts on this?